Add files via upload

Evgeniy997 opened the pull request:
Проверь пожалуйста , всё правильно?

Evgeniy997 committed:
Add files via upload

isys35 commented:
Для домашки нужно создать папку и положить домашку внутрь. Так же нужен код с 2-домашки.

Evgeniy997 committed:
Create Евгений Лешок

Evgeniy997 committed:
Add files via upload

Evgeniy997 committed:
Add files via upload

Выполненное задание

Evgeniy997 committed:
Merge pull request #21 from isys35/Evgeniy997-patch-1

Evgeniy997 patch 1

Evgeniy997 committed:
Add files via upload

Выполненное домашнее задание

Evgeniy997 committed:
Merge pull request #23 from isys35/leshok-homework4

Leshok homework5

Evgeniy997 committed:
Merge pull request #24 from isys35/Liashok-homework5

Liashok homework5

Evgeniy997 committed:
для проверки

Дмитрий , добрый день есть такой вопрос  как я могу запустить данный бот, и что я сделал не так ? можно узнать ?

Evgeniy997 committed:
Merge pull request #25 from isys35/Evgeniy-leshok-bot

для проверки
